en-US/social/2017/paperstorm.lang

```diff
@@ -0,0 +1,90 @@
+## NOTE: Strings used as tweets and Facebook posts. Some strings at the bottom of the file are part of a short GIF/video to be shared on social media.
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;Email not getting your MEP’s attention? Try dropping a flyer (or 10) on their town office. Help #fixcopyright with paperstorm.it
+Email not getting your MEP’s attention? Try dropping a flyer (or 10) on their town office. Help #fixcopyright with paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;One way to get EU lawmakers’ attention: drop thousands of digital leaflets. Help us #fixcopyright paperstorm.it
+One way to get EU lawmakers’ attention: drop thousands of digital leaflets. Help us #fixcopyright paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;Yes! We dropped 10 000 000 leaflets on the EU Parliament! 🎉 We think [@comodinicachia] noticed :) Join paperstorm.it!
+Yes! We dropped 10 000 000 leaflets on the EU Parliament! 🎉 We think [@comodinicachia] noticed :) Join paperstorm.it!
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;Props to [@name]! High score of [number] #paperstorm leaflets dropped! 💪 Are you a copyfighter? It’s your turn. paperstorm.it
+Props to [@name]! High score of [number] #paperstorm leaflets dropped! 💪 Are you a copyfighter? It’s your turn. paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;OMG [@JeanMarieCAVADA] the Eiffel tower is covered in a #paperstorm. [number] leaflets have been dropped @ paperstorm.it
+OMG [@JeanMarieCAVADA] the Eiffel tower is covered in a #paperstorm. [number] leaflets have been dropped @ paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;CLICK FOR EUROPE. Join the #paperstorm & demand Europe’s politicians #fixcopyright. Drop virtual leaflets at paperstorm.it
+CLICK FOR EUROPE. Join the #paperstorm & demand Europe’s politicians #fixcopyright. Drop virtual leaflets at paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;Clicks for europe. Clicks for copyright. Clicks for culture. Join the #paperstorm and demand #copyright reform in Europe 🇪🇺.
+Clicks for europe. Clicks for copyright. Clicks for culture. Join the #paperstorm and demand #copyright reform in Europe 🇪🇺.
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
```

Comment:
Is `[The Palace of Science and Culture]` a link or supposed to stay untranslated?